src/wrap/mod.rs

```diff
@@ -42,33 +42,56 @@ impl Checker for Wrapper {
             SuggestionSet::new(),
             |mut acc, (origin, chunks)| {
                 for chunk in chunks {
-                    let mut new_lines = chunk
+                    let mut new_lines = dbg!(&chunk)
                         .as_str()
                         .split("\n\n")
                         .collect::<Vec<&str>>()
                         .iter()
-                        .map(|s| s.replace("\n", "") )
+                        .map(|s| s.replace("\n", ""))
                         .fold::<Vec<String>, _>(Vec::new(), |mut acc, comment| {
                             let mut new_comment = wrapper
                                 .wrap_iter(comment.trim())
-                                .map(|b| b.into_owned()).collect();
+                                .map(|b| b.into_owned())
+                                .collect();
                             acc.append(&mut new_comment);
                             acc.push("".into());
                             acc
                         });
                     // remove last newline
                     let _ = new_lines.pop();
+                    // @todo that's too easy
```

Comment:
can you elaborate on a case where this is not good enough?

Reply:
At some point I had an example which did not work, unfortunately I didn't investigate at that time.. The todo is mostly for me to look at it at some point

Reply:
Actually:
```text
V eeeeeeeeeeeeeeeeeeeeeeeeeee rrrrrrrrrrrrrrrrrrrrrrrr y long
shorty.
```

becomes

```text
V eeeeeeeeeeeeeeeeeeeeeeeeeee 
rrrrrrrrrrrrrrrrrrrrrrrr y long shorty.
```

which would have the same number of lines.

I think it'd be best to just blindly compare the full `str` repr and avoid any false positives and be done with it :)